src/main/kotlin/three/consulting/epoc/dto/ProjectDTO.kt

```diff
@@ -0,0 +1,29 @@
+package three.consulting.epoc.dto
+
+import three.consulting.epoc.entity.Project
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class ProjectDTO(
+    val id: Long? = null,
+    val name: String,
+    val description: String? = null,
+    val startingDate: LocalDate,
+    val endDate: LocalDate? = null,
+    val customer: CustomerDTO? = null,
+    val managingEmployee: EmployeeDTO? = null,
```

Comment:
These shouldn't be nullable as we don't want them to be nullable in the database either.